torch/_torch_docs.py

```diff
@@ -4365,6 +4365,48 @@ def merge_dicts(*dicts):
             [1.0311, 0.3901, 0.5049]])
 """)
 
+add_docstr(torch.movedim,
```

Comment:
We should add this to the catalogue of view ops: https://pytorch.org/docs/stable/tensor_view.html